go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/PuerkitoBio/purell v1.1.1 // indirect
 	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/blang/semver v3.5.1+incompatible // indirect
```

Comment:
In the L42, latest version of github.com/blang/semver is already added as indirect dependency. So you can remove this. Same for go.sum file as well.

Reply:
Solved.